lib/fpm/cookery/utils.rb

```diff
@@ -85,6 +85,12 @@ def patch(src, level = 0)
         raise "#{src} does not exist" unless File.exist? src
         safesystem "patch -p#{level} --batch < #{src}"
       end
+
+      def go(*args)
+        args = argument_build(*args)
+        args.each {|a| a == '--mod=vendor' && ENV['GO111MODULE'] = 'on' }
```

Comment:
Previously I had this line like:

```
safesystem "go", *args
```
but would end up with the following error:

```
fpm-cookery-0.33.0/lib/fpm/cookery/utils.rb:10:in `system': no implicit conversion of Symbol into String (TypeError)
```

So the join is my crappy solution. Happy for other suggested workarounds.